diagnostic.md

```diff
@@ -35,5 +39,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each .phonenumber as |phoneNum|}}
+      {{contact/my-contact/my-phone phoneNum=phoneNum}}
+    {{/each}}
```

Comment:
We need to access the object property via `contact.phones` instead of `.phonenumber`.  You don't need `contact/my-contact/`.  So it should look something like this:
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```